src/features/auth/useWalletAuth.ts

```diff
@@ -21,12 +32,28 @@ const updateToken = ({ address, authTokens }: IAuth) => {
   setAuthToken(token);
 };
 
+const AUTH_STORAGE_KEY = 'capeAuth';
+
+const saveAuth = (auth: IAuth) =>
+  localStorage.setItem(AUTH_STORAGE_KEY, JSON.stringify(auth));
+const getSavedAuth = (): IAuth => {
+  try {
+    const auth = localStorage.getItem(AUTH_STORAGE_KEY);
+    assert(auth);
```

Comment:
assert is just here to verify the fetched local storage auth data object is not an empty {} ?

JSON.parse would otherwise accept and return it even though there's no auth data there?

Reply:
oh, this is leftover from when i removed assertDef a few months ago: 7fe85945

looking at this more closely now, it's true we could drop the assert, since the try/catch will handle all the weird cases.